README.md

```diff
@@ -32,7 +32,7 @@ This role requires root access, so either configure it in your inventory files,
 - hosts: dnsserver
   become: true
   roles:
-    - role: aalaesar.install_nextcloud
+    - role: aalaesar.nextcloud
```

Comment:
seems the name change was also commited in this branch.
can you revert it ? thanks :)

Reply:
@aalaesar this PR was synced automatically after the merge please check again.